pkg/osutils/osutils.go

```diff
@@ -122,4 +122,4 @@ func Kubectl(arguments []string) error {
 	}
 
 	return nil
-}
+}
```

Comment:
There should be a newline at the end of this file, but more importantly it looks like nothing was actually changed here.